### ISSUE TITLE
Accessibility: MenuItem anchor has tabIndex of -1

### ISSUE THREAD

**Issue by rygine** [react-bootstrap/react-bootstrap#231]
I'm not sure why the tabIndex is being forced to -1, but menu items are not accessible via keyboard navigation. Simply removing the tabIndex fixes the problem.

**Comment by dozoisch** [comment]
Accessibility => putting in 1.0.0

**Comment by mtscout6** [comment]
Related to #526

**Comment by taion** [comment]
This was actually resolved by #526 - menus are keyboard accessible, just not via tab.
